src/Cabal2Nix/CorePackages.hs

```diff
@@ -8,7 +10,7 @@ module Cabal2Nix.CorePackages ( corePackages, coreBuildTools ) where
 -- The commented packages have previously been non-core, so we don't filter
 -- them.
 corePackages :: [String]
-corePackages = [
+corePackages = map toNixName [
```

Comment:
What is the purpose of this change? Currently, `corePackages` contain a list of Haskell package names, *not* Nix attributes. Since the latter can be generated out of the former -- but not vice versa -- it seems like a bad idea to apply `toNixName` here.